package.json

```diff
@@ -24,6 +24,8 @@
     "dev": "lerna run --parallel --scope @firebase/* --scope firebase --scope rxfire dev",
     "build": "lerna run --scope @firebase/* --scope firebase --scope rxfire prepare",
     "build:exp": "lerna run --scope @firebase/*-exp --scope firebase-exp build",
+    "build:release": "lerna run --scope @firebase/* --scope firebase --ignore @firebase/*-exp --scope firebase-exp build",
+    "build:exp:release": "yarn build:exp && yarn --cwd packages-exp/app-exp typings:public",
```

Comment:
I'd like to keep the swapping typings step in `@firebase/app-exp` because I want this step to run when I run `build:release` in `@firebase/app-exp`.
How about `"build:exp:release": "lerna run --scope @firebase/*-exp --scope firebase-exp prepare"`?

Reply:
Ok, app-exp `prepare`, which is only run by the root script, will not swap typings, but app-exp `build:release` which is only run locally, will.